frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -0,0 +1,185 @@
+<template lang='pug'>
+  .settings-container
+    section.card
+      .c-header
+        fieldset.c-filters
+          legend.sr-only Visible logs
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='form.filter' value='error')
+            i18n Errors
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='form.filter' value='warn')
+            i18n Warnings
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='form.filter' value='debug')
+            i18n Debug
```

Comment:
Missing checkbox for `Info` and `Log`

Reply:
Ups! 💣 `console.log` can't be captured - it will start a loop of logs. because each time we capture a log, sbp gets called (events.emit CAPTURED_LOGS), which logs the event itself, causing a new `console.log` to be captured and so on...... 👀

Reply:
Nice catch. add the offending event-related selector to `selBlacklist` in `main.js`. Or you can even blacklist the entire `okTurtles.events` domain via `domainBlacklist`)

Reply:
Oh, and whatever you choose, please make sure to add a comment explaining why we're adding it to the blacklist. Otherwise we'll forget.

Reply:
So... by adding `okTurtles.events/emit` to the blacklist, it won't log any event that happens. Not sure if that's okay.... the logs are quite useful to understand the flow of this.

Reply:
True. Let me think.

Reply:
Figured it out.

Instead of: `sbp('okTurtles.events/emit', CAPTURED_LOGS, lastEntry)`

Do this:

```js
sbp('sbp/selectors/fn', 'okTurtles.events/emit')(CAPTURED_LOGS, lastEntry)
```

😄

Reply:
Make sure to add a comment explaining anything weird.

```js
// To avoid infinite loop because we log all selector calls, we run sbp calls here
// in a roundabout way by getting the function to which they're mapped
sbp('sbp/selectors/fn', 'okTurtles.events/emit')(CAPTURED_LOGS, lastEntry)
```

Reply:
Do you mind explaining that magic🔮?

Reply:
heh, we posted at the same time :)

See explanation above :)

Reply:
ohhh those ugly workarounds 🙈. Pushed.